src/components/profile/Recovery/RecoveryContainer.js

```diff
@@ -89,27 +88,31 @@ class RecoveryContainer extends Component {
         resendingLink: ''
     };
 
+    componentDidMount = () => {
+        this.props.loadRecoveryMethods()
+    }
+
     handleEnableMethod = (method) => {
-        const { history, accountId } = this.props;
+        const { history, account: { accountId } } = this.props;
 
         history.push(`${method !== 'phrase' ? '/set-recovery/' : '/setup-seed-phrase/'}${accountId}`);
     }
 
     handleDeleteMethod = (method) => {
-        const { deleteRecoveryMethod, loadRecoveryMethods, accountId } = this.props;
+        const { deleteRecoveryMethod, loadRecoveryMethods } = this.props;
 
         this.setState({ deletingMethod: method.kind })
         deleteRecoveryMethod(method)
             .then(({ error }) => {
                 if (error) return
-                loadRecoveryMethods(accountId);
```

Comment:
did this get changed to load recovery methods for current account? not sure how will this play with https://github.com/near/near-wallet/issues/580

Reply:
`accountId` param was unnecessary, `loadRecoveryMethods` was using current logged account always:
https://github.com/near/near-wallet/pull/534/files/21a8e48a3f8fac0ad033d5640c34a5e72bfe8c21#diff-b3e6d1b5726af892a08cb5d16f807a6cL17